doc/usr/source/9_other/8_interpreter.rst

```diff
@@ -33,7 +33,7 @@ The overall structure is as follows:
     {
       "var1": "24",
       "var2": false,
-      "var3": "0.5"
+      "var3": "1/2"
```

Comment:
Shouldn't this be `1.0/2.0`?

Reply:
We were supporting both in the CSV format, so I kept both in the JSON format.

Reply:
I see. I prefer consistency with Lustre though.

Reply:
Ok. I will update the documentation after the merge.